### DISCUSSION TITLE
df.to_xyz - yes please, or good alternative :-)

### DISCUSSION THREAD

**Discussion by Hendrik1987** [DHI/mikeio#563] [Ideas]
I use this all the time to quickly get a xyz formatted such that it is accepted by MIKE.
https://github.com/DHI/mikeio/blob/887b0761840fa798f98cd9de0cfa4dff22895c23/mikeio/xyz.py#L26

So please keep this feature or an as simple alternative.

Validation could be a nice additon!
https://github.com/DHI/mikeio/blob/887b0761840fa798f98cd9de0cfa4dff22895c23/mikeio/xyz.py#L22

**Comment by ecomodeller** [comment]
We'll keep it!
